mindmeld/query_cache.py

```diff
@@ -108,22 +109,19 @@ def __del__(self):
 
         self.disk_connection.close()
 
-    def compatible_version(self):
+    def compatible_version_or_text_prep_pipeline(self):
```

Comment:
Let's revert this name back since we don't have any knowledge of the text_prep_pipeline in this module anymore.

Reply:
Fixed